pkg/quarkus/v1alpha/scaffolds/internal/templates/makefile.go

```diff
@@ -91,6 +101,17 @@ docker-build: ## Build docker image with the manager.
 docker-push: ## Push docker image with the manager.
 	mvn package -Dquarkus.container-image.push=true -Dquarkus.container-image.image=${IMG}
 
+##@Bundle
+bundle-generate:  
+	cat target/kubernetes/*.{{ .Resource.QualifiedGroup }}-{{ .Resource.Version }}.yml target/kubernetes/kubernetes.yml | operator-sdk generate bundle -q --overwrite --version 0.1.1 --default-channel=stable --channels=stable --package=memcached-quarkus-operator
+	operator-sdk bundle validate ./bundle
+
+bundle-build:
```

Comment:
```suggestion
.PHONY: bundle-build
bundle-build: ## Build the bundle image.
```

We want `make help` to show this target

Reply:
Updated according to the above comments